src/plugins/calendar/_base.scss

```diff
@@ -132,7 +132,6 @@
 .cal-days {
 	a {
 		@extend %cal-days-a-div;
-		color: $cal-days-link;
```

Comment:
It was declared, but wasn't actually used. The links were overrode by the styles from `.cal-wd`. I think the entire calendar plugin needs to be reviewed independently in another PR like the tabs plugin. For this PR, I was thinking of just keeping the variables that were there already that were actively used by the plugin.

![screen shot 2015-02-13 at 5 55 47 am](https://cloud.githubusercontent.com/assets/6571996/6188485/6b5685a6-b346-11e4-8a27-0d1abfce9e87.png)